sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -78,6 +78,16 @@ select timestamp'2019-10-06 10:11:12.345678' - date'2020-01-01';
 select timestamp'2019-10-06 10:11:12.345678' - null;
 select null - timestamp'2019-10-06 10:11:12.345678';
 
+-- subtract timestamps without time zone
+select date'2020-01-01' - to_timestamp_ntz('2019-10-06 10:11:12.345678');
+select to_timestamp_ntz('2019-10-06 10:11:12.345678') - date'2020-01-01';
+select to_timestamp_ntz('2019-10-06 10:11:12.345678') - null;
+select null - to_timestamp_ntz('2019-10-06 10:11:12.345678');
+select to_timestamp_ntz('2019-10-07 10:11:12.345678') - to_timestamp_ntz('2019-10-06 10:11:12.345677');
+select to_timestamp_ntz('2019-10-06 10:11:12.345677') - to_timestamp_ntz('2019-10-07 10:11:12.345678');
+select to_timestamp_ntz('2019-10-07 10:11:12.345678') - timestamp'2019-10-06 10:11:12.345678';
+select timestamp'2019-10-06 10:11:12.345678' - to_timestamp_ntz('2019-10-07 10:11:12.345678');
```

Comment:
let's use `to_timestamp` in the test, in case we make `timestamp'xxx'` return TIMESTAMP NTZ later.